openstack/loadbalancer/v2/pools/requests.go

```diff
@@ -18,17 +18,22 @@ type ListOptsBuilder interface {
 // sort by a particular Pool attribute. SortDir sets the direction, and is
 // either `asc' or `desc'. Marker and Limit are used for pagination.
 type ListOpts struct {
-	LBMethod       string `q:"lb_algorithm"`
-	Protocol       string `q:"protocol"`
-	ProjectID      string `q:"project_id"`
-	AdminStateUp   *bool  `q:"admin_state_up"`
-	Name           string `q:"name"`
-	ID             string `q:"id"`
-	LoadbalancerID string `q:"loadbalancer_id"`
-	Limit          int    `q:"limit"`
-	Marker         string `q:"marker"`
-	SortKey        string `q:"sort_key"`
-	SortDir        string `q:"sort_dir"`
+	LBMethod          string `q:"lb_algorithm"`
```

Comment:
did you miss `AlpnProtocols` here?

Reply:
```AlpnProtocols``` is an array. Didn't think you could sort an array?